test/parallel/test-https-localaddress.js

```diff
@@ -21,19 +21,19 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
 if (!common.hasMultiLocalhost())
   common.skip('platform-specific test.');
 
-const fs = require('fs');
 const assert = require('assert');
 const https = require('https');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readSync('keys/agent1-key.pem'),
```

Comment:
This could be using method `fixtures.readKey` instead of `fixtures.readSync` to get the keys.